odalic/src/main/java/cz/cuni/mff/xrg/odalic/input/ListsBackedInputBuilder.java

```diff
@@ -20,6 +20,17 @@
   
   public ListsBackedInputBuilder() { }
   
+  public void setInitialInput(Input initialInput) {
```

Comment:
I think this method should become an optional constructor instead, as it does not make much sense to call it any other time than as the first setting method. I will fix that myself.